Description: Update the CMakeToolchain.txt inline with minkipc
Author: Abhinaba Rakshit <abhinaba.rakshit@oss.qualcomm.com>
Origin: other, Qualcomm forked mink specific changes over OPTEE-Test
Last-Update: 2026-07-14
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: optee_test/CMakeToolchain.txt
===================================================================
--- optee_test.orig/CMakeToolchain.txt
+++ optee_test/CMakeToolchain.txt
@@ -1,3 +1,5 @@
-set (CMAKE_SYSTEM_NAME Linux)
+set(CMAKE_SYSTEM_NAME Linux)
+set(CMAKE_SYSTEM_PROCESSOR aarch64)
 
-set (CMAKE_SYSTEM_PROCESSOR arm)
+set(CMAKE_C_COMPILER aarch64-linux-gnu-gcc)
+set(CMAKE_CXX_COMPILER aarch64-linux-gnu-g++)
